deployment_script.py

```diff
@@ -1,11 +1,11 @@
 from pymongo import MongoClient
 from decouple import config
-from text_based_scenarios._0_1_10_participant_log import main as ingest_participant_log
+from text_based_scenarios.convert_yaml_to_json_config import main as generate_text_based_configs
 VERSION_COLLECTION = "itm_version"
 MONGO_URL = config('MONGO_URL')
 
 # Change this version if running a new deploy script
-db_version = "0.1.10"
+db_version = "0.1.11"
```

Comment:
This won't work on the version check, the strings won't think it is larger, we had to do a hack for the 0.1.10, make this one 0.2.0 (my script is going to be 0.2.1)